linkerd/protocol/h2/src/test/scala/io/buoyant/linkerd/protocol/h2/H2AddForwardedHeaderConfigTest.scala

```diff
@@ -0,0 +1,150 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.fasterxml.jackson.databind.exc.InvalidTypeIdException
+import com.twitter.finagle.Stack
+import io.buoyant.config.Parser
+import io.buoyant.router.RouterLabel
+import io.buoyant.router.http.{AddForwardedHeaderConfig, ForwardedHeaderLabeler}
+import io.buoyant.test.FunSuite
+
+class H2AddForwardedHeaderConfigTest extends FunSuite {
```

Comment:
Is any of this H2 specific?  Isn't this just tests of the AddForwardedHeaderConfig, which lives in http-base?

Reply:
That is correct, thanks for catching that. It is definitely redundant to have to test for this all over again.